docs/customizing.md

```diff
@@ -138,7 +144,7 @@ Recommended Workflows for Pulpcore & Plugin Versioning
 Initial installation:
 
 1. Make sure you are running the latest version of the installer, which installs pulpcore's
-   latest version.
+   latest 3.y release.
```

Comment:
We need an entire new section under `Recommended Workflows for Pulpcore & Plugin Versioning`.

One named like "latest version on minor release branches".

It will be for both pulpcore and the plugins.

Reply:
I added a new section